noxfile.py

```diff
@@ -61,7 +61,6 @@ def docs(session: nox.Session) -> None:
 
     if session.posargs:
         if "--serve" in session.posargs:
-            print("Launching docs at http://localhost:8000/ - use Ctrl-C to quit")
```

Comment:
Oh, wow, auto fixing no print statements might be something to turn off. :)

Reply:
Huh that seems a bit aggressive!